src/lib/containers/entity/page/breadcrumbs/EntityPageBreadcrumb.stories.tsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react'
+import { ComponentStory, ComponentMeta } from '@storybook/react'
+
+import EntityPageBreadcrumbs from './EntityPageBreadcrumbs'
+
+// More on default export: https://storybook.js.org/docs/react/writing-stories/introduction#default-export
+export default {
+  title: 'Synapse/EntityPage/Breadcrumb',
+  component: EntityPageBreadcrumbs,
+  // More on argTypes: https://storybook.js.org/docs/react/api/argtypes
+  argTypes: {},
+} as ComponentMeta<typeof EntityPageBreadcrumbs>
+
+// More on component templates: https://storybook.js.org/docs/react/writing-stories/introduction#using-args
+const Template: ComponentStory<typeof EntityPageBreadcrumbs> = args => (
+  <>
+    {/* Add some space above the component so the tooltip positioning is more accurate */}
```

Comment:
Is this something that should be in the EntityPageBreadcrumbs style, or is it specific to the Storybook environment?

Reply:
Specific to Storybook; the tooltip appears below the element just because Storybook doesn't give it the necessary space above.